packages/happy-dom/src/nodes/html-input-element/HTMLInputElement.ts

```diff
@@ -558,6 +558,24 @@ export default class HTMLInputElement extends HTMLElement implements IHTMLInputE
 		if (!checked) {
 			this.removeAttributeNS(null, 'checked');
 		} else {
+			// Need to remove all other radio buttons checked with the same name
+			const name = this.getAttributeNS(null, 'name');
```

Comment:
I'm working on a PR with a lot of Form/Input element related things, including fixing the "checked" property:
https://github.com/capricorn86/happy-dom/pull/677/files#diff-bb649479cc3389b9fff9883a1cc18a2b9edc08aac12cb0689ecabf13f21be16bL557

I have been working on it for quite some time, but my goal is to release it this week.

Reply:
The PR contains so many fixes that it might break the UI of Github. In that case you can press "." on the keyboard to open the online editor to view it.